src/ui/utils/apolloClient.ts

```diff
@@ -2,10 +2,14 @@ import { ApolloClient, InMemoryCache, NormalizedCacheObject } from "@apollo/clie
 import { HttpLink } from "apollo-link-http";
 import { DocumentNode } from "graphql";
 import { assert } from "protocol/utils";
+import { getTest } from "ui/utils/environment";
 
 let apolloClient: ApolloClient<NormalizedCacheObject> | null = null;
 
 export function query({ variables = {}, query }: { variables: any; query: DocumentNode }) {
+  if (!getTest()) {
```

Comment:
The intention was not to disable some graphql queries in tests but to to always seek to the endpoint in tests to make them more predictable.

Reply:
That makes sense